test/framework/thanosruler.go

```diff
@@ -40,6 +40,7 @@ func (f *Framework) MakeBasicThanosRuler(name string, replicas int32, queryEndpo
 			Replicas:       &replicas,
 			QueryEndpoints: []string{queryEndpoint},
 			LogLevel:       "debug",
+			RuleSelector:   &metav1.LabelSelector{},
```

Comment:
now that MakeBasicThanosRuler() returns a Thanos Ruler that selects all rules in the current namespace, we can remove L68 in test/e2e/thanosruler_test.go?

Reply:
Indeed!